### PR TITLE
Update customize-exploit-protection.md

### PR DESCRIPTION
**Pull request by msbemba**
Removed Borken Checkmark Yes/No in the document as the image source is not available in media

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5125

File: microsoft-365/security/defender-endpoint/customize-exploit-protection.md [hunk 1]
```diff
@@ -52,27 +52,27 @@ For the associated PowerShell cmdlets for each mitigation, see the [PowerShell r
...
+| Force randomization for images (Mandatory ASLR) | Forcibly relocates images not compiled with /DYNAMICBASE. Can optionally fail loading images that don't have relocation information. | System and app-level  | No |
+| Randomize memory allocations (Bottom-Up ASLR) | Randomizes locations for virtual memory allocations. It includes system structure heaps, stacks, TEBs, and PEBs. Can optionally use a wider randomization variance for 64-bit processes. | System and app-level | No |
+| Validate exception chains (SEHOP) | Ensures the integrity of an exception chain during exception dispatch. Only configurable for 32-bit (x86) applications. | System and app-level  | No |
+| Validate heap integrity | Terminates a process when heap corruption is detected. | System and app-level  | No |
+| Arbitrary code guard (ACG) | Prevents the introduction of non-image-backed executable code and prevents code pages from being modified. Can optionally allow thread opt-out and allow remote downgrade (configurable only with PowerShell). | App-level only | Yes |
+| Block low integrity images | Prevents the loading of images marked with Low Integrity. | App-level only | !Yes |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/customize-exploit-protection.md]:
```suggestion
| Block low integrity images | Prevents the loading of images marked with Low Integrity. | App-level only | Yes |
```

File: microsoft-365/security/defender-endpoint/customize-exploit-protection.md [hunk 2]
```diff
@@ -52,27 +52,27 @@ For the associated PowerShell cmdlets for each mitigation, see the [PowerShell r
...
+| Code integrity guard | Restricts loading of images signed by Microsoft, WHQL, or higher. Can optionally allow Microsoft Store signed images. | App-level only | Yes |
+| Disable extension points | Disables various extensibility mechanisms that allow DLL injection into all processes, such as AppInit DLLs, window hooks, and Winsock service providers. | App-level only | No |
+| Disable Win32k system calls | Prevents an app from using the Win32k system call table. | App-level only | Yes |
+| Don't allow child processes | Prevents an app from creating child processes. | App-level only | Yes |
+| Export address filtering (EAF) | Detects dangerous operations being resolved by malicious code. Can optionally validate access by modules commonly used by exploits. | App-level only | Yes |
+| Import address filtering (IAF) | Detects dangerous operations being resolved by malicious code. | App-level only | Yes |
+| Simulate execution (SimExec) | Ensures that calls to sensitive APIs return to legitimate callers. Only configurable for 32-bit (x86) applications. Not compatible with ACG | App-level only | Yes |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/customize-exploit-protection.md]:
```suggestion
| Simulate execution (SimExec) | Ensures that calls to sensitive APIs return to legitimate callers. Only configurable for 32-bit (x86) applications. Not compatible with ACG. | App-level only | Yes |
```

File: microsoft-365/security/defender-endpoint/customize-exploit-protection.md [hunk 3]
```diff
@@ -52,27 +52,27 @@ For the associated PowerShell cmdlets for each mitigation, see the [PowerShell r
...
+| Validate API invocation (CallerCheck) | Ensures that sensitive APIs are invoked by legitimate callers. Only configurable for 32-bit (x86) applications. Not compatible with ACG | App-level only | Yes |
+| Validate handle usage | Causes an exception to be raised on any invalid handle references. | App-level only | No |
+| Validate image dependency integrity | Enforces code signing for Windows image dependency loading. | App-level only | No |
+| Validate stack integrity (StackPivot) | Ensures that the stack hasn't been redirected for sensitive APIs. Not compatible with ACG | App-level only | Yes |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/customize-exploit-protection.md]:
```suggestion
| Validate stack integrity (StackPivot) | Ensures that the stack hasn't been redirected for sensitive APIs. Not compatible with ACG. | App-level only | Yes |
```

File: microsoft-365/security/defender-endpoint/customize-exploit-protection.md [hunk 4]
```diff
@@ -81,10 +81,10 @@ For the associated PowerShell cmdlets for each mitigation, see the [PowerShell r
 >
 > | Enabled in **Program settings** | Enabled in **System settings** | Behavior |
 > | ------------------------------- | ------------------------------ | -------- |
-> | ![Check mark yes](/security/defender-endpoint/images/svg/check-yes) | ![Check mark no](/security/defender-endpoint/images/svg/check-no) | As defined in **Program settings** |
-> | ![Check mark yes](/security/defender-endpoint/images/svg/check-yes) | ![Check mark yes](/security/defender-endpoint/images/svg/check-yes) | As defined in **Program settings** |
-> | ![Check mark no](/security/defender-endpoint/images/svg/check-no) | ![Check mark yes](/security/defender-endpoint/images/svg/check-yes) | As defined in **System settings** |
-> | ![Check mark no](/security/defender-endpoint/images/svg/check-no) | ![Check mark yes](/security/defender-endpoint/images/svg/check-yes) | Default as defined in **Use default** option |
+> |         Yes        |        No       | As defined in **Program settings** |
+> |         Yes        |        No       | As defined in **Program settings** |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/customize-exploit-protection.md]: Previously Yes - Yes, are these values correct?

File: microsoft-365/security/defender-endpoint/customize-exploit-protection.md [hunk 5]
```diff
@@ -81,10 +81,10 @@ For the associated PowerShell cmdlets for each mitigation, see the [PowerShell r
...
+> |         No         |        No       | As defined in **System settings** |
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/customize-exploit-protection.md]: Previously No - Yes, are these values correct?

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/security/defender-endpoint/customize-exploit-protection.md (hunk 5 above)
Comment (msbemba) [reply microsoft-365/security/defender-endpoint/customize-exploit-protection.md]: That is correct

File: microsoft-365/security/defender-endpoint/customize-exploit-protection.md (hunk 4 above)
Comment (msbemba) [reply microsoft-365/security/defender-endpoint/customize-exploit-protection.md]: Yes

Comment (msbemba) [reply microsoft-365/security/defender-endpoint/customize-exploit-protection.md]: I'll check and fix this

File: microsoft-365/security/defender-endpoint/customize-exploit-protection.md (hunk 5 above)
Comment (msbemba) [reply microsoft-365/security/defender-endpoint/customize-exploit-protection.md]: I'll check and fix this

File: microsoft-365/security/defender-endpoint/customize-exploit-protection.md (hunk 4 above)
Comment (msbemba) [reply microsoft-365/security/defender-endpoint/customize-exploit-protection.md]: Fixed. Please review

Comment (JohanFreelancer9) [reply microsoft-365/security/defender-endpoint/customize-exploit-protection.md]: @msbemba - thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Dansimp Please merge this PR. Thanks

**Pull request merged**